entities/Rules/highestRollingPlayerGoesFirst.js

```diff
@@ -19,4 +20,8 @@ module.exports = function highestRollingPlayerGoesFirst({ UI }, { players }) {
   // TODO: Handle Ties
   let shiftValue = (players.length - highest) % players.length;
   for (; shiftValue > 0; --shiftValue) players.push(players.shift());
+
+  // add ids to players
+  for (let id = 0; id < players.length; id++) players[id].id = id;
```

Comment:
```javascript
players.forEach((player, index) => player.id = index);
```
?

Reply:
ahh, yes. :) I knew there was a better way at the time and couldn't think of it.